env_build_commands.md

```diff
@@ -0,0 +1,26 @@
+# Instructions for building the CNN_MASKING environment that works on modern GPUs
```

Comment:
Ryan, this file won't work. Can you environmentalize this sequence of conda and pip packages?

Reply:
I will do more research on this, and let you know. My understanding is of the basic format of the conda env yml file, but maybe there is a way. I will let you know.

Reply:
Resolved via this comment: https://github.com/pnlbwh/CNN-Diffusion-MRIBrain-Segmentation/pull/40#issuecomment-2026111530